packages/dashboard/src/pages/SetupPersona/Onboarding/index.tsx

```diff
@@ -0,0 +1,179 @@
+        width: 190,
+        height: 36,
+        borderRadius: 99,
+        marginLeft: 42,
+    },
+    plugins: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        width: 36,
+        height: 36,
+        borderRadius: 99,
+        // hard color
+        background: '#F0F0F4',
+        marginLeft: 18,
+        marginRight: 18,
+    },
+    more: {
+        transform: 'rotate(90deg)',
+    },
+    pinCard: {
+        marginTop: 18,
+        borderRadius: 8,
+        border: `1px solid ${theme.palette.maskColor.line}`,
+        background: theme.palette.maskColor.bottom,
+        padding: 16,
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+    },
+    trend: {
+        position: 'fixed',
+        top: 206,
+        right: 408,
+    },
+    typed: {
+        fontSize: 36,
+        lineHeight: 1.2,
+        fontWeight: 700,
+        fontFamily: 'Consolas, Monaco',
+        '& > strong': {
+            color: theme.palette.maskColor.highlight,
+        },
+    },
+    line: {
+        width: '21em',
+        borderRight: '1px solid transparent',
+        animation: 'typing 3.5s steps(42, end)',
+        wordBreak: 'break-all',
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        '@keyframes typing': {
+            from: { width: 0 },
+            to: { width: '21em' },
+        },
+    },
+    endTyping: {
+        opacity: 0.5,
+    },
+    twitter: {
+        color: theme.palette.maskColor.bottom,
+    },
+}))
+
+export const Onboarding = memo(function Onboarding() {
+    const t = useDashboardI18N()
+    const { classes, cx } = useStyles()
+    const [current, setCurrent] = useState(0)
+
+    useEffect(() => {
+        const clean = setInterval(() => {
+            setCurrent((prev) => {
+                if (prev < 3) return prev + 1
+                console.log(clean)
+                clearInterval(clean)
+                return prev
+            })
+        }, 4000)
+
+        return () => clearInterval(clean)
+    }, [])
+
+    const words = useMemo(() => {
+        return [
+            <DashboardTrans.persona_onboarding_creating_identity key={0} components={{ strong: <strong /> }} />,
+            <DashboardTrans.persona_onboarding_generating_accounts key={1} components={{ strong: <strong /> }} />,
+            <DashboardTrans.persona_onboarding_encrypting_data key={2} components={{ strong: <strong /> }} />,
+            <DashboardTrans.persona_onboarding_ready key={3} components={{ strong: <strong /> }} />,
+        ]
+    }, [])
+
+    const onSetupTwitter = useCallback(async () => {
+        const url = await Services.SocialNetwork.setupSite(EnhanceableSite.Twitter, false)
+        if (!url) return
+        await delay(300)
+        browser.tabs.create({
+            active: true,
+            url,
+        })
+        window.close()
```

Comment:
why not just navigate?

Reply:
I think we need to wait for the designer's confirmation.